Sources/Valkey/RESP/RESPDecodeError.swift

```diff
@@ -65,11 +65,20 @@ public struct RESPDecodeError: Error {
         }
     }
     /// Does not match the expected array size
-    public static func invalidArraySize(_ array: RESPToken.Array, expectedSize: Int) -> Self {
-        .init(
+    public static func invalidArraySize(_ array: RESPToken.Array, expectedSize: Int = -1, minExpectedSize: Int = -1) -> Self {
```

Comment:
Instead of using `-1` as an indication the expectedSize/minExpectedSize is not used make them optional